examples/intraday_examples/lead_time_analysis.py

```diff
@@ -0,0 +1,312 @@
+        }
+    }
+    return curve_dict[benchmark_forecast], curve_dict[intraday_forecast]
+
+
+def getkey() -> Session:
+    """This function configures Session from volue_insight_timeseries with local environment variable "WAPI_INI_READ" to allow for connection to WAPI.
+
+    Returns:
+        session (volue_insight_timeseries.session.Session): A session object that can communicate with WAPI.
+    """
+    config = os.getenv('WAPI_INI_READ')
+    session = Session(config_file=config)
+    return session
+
+
+def get_instance_curve(session: Session, curve_name: str, issue_date: str) -> pd.Series:
+    """ This function retrieves the instance curve released at the determined issue date.
+    Args:
+    session: Session object to interact with WAPI.
+    curve_name: The name of the curve that is to be retrieved as an instance curve.
+    issue_date: The user defined date of issue for the instance curve.
+
+    Returns:
+        time_series_pandas: Curve data as pandas time series.
+    """
+    instance_curve = session.get_curve(name=curve_name)
+    time_series = instance_curve.get_instance(issue_date=issue_date)
+    time_series_pandas = time_series.to_pandas()
+    return time_series_pandas
+
+
+def get_absolute_curve(data_time: pd.Timestamp, session: Session, curve_name: str,
+                       delta_hours: pd.Timedelta) -> pd.Series:
+    """This function retrieves the curve which displays how the forecasted value at timestamp "data_time" develops with each release (absolute curve).
+
+    Args:
+        data_time: The timestamp of the forecasted value.
+        session: Session object to interact with WAPI.
+        curve_name: The name of the curve that is to be retrieved as an absolute curve.
+        delta_hours: A value that defines how many hours back from the timestamp of data_time to retrieve data.
+
+    Returns:
+        time_series_pandas: Absolute curve data as pandas time series.
+    """
+
+    date_from = data_time - delta_hours
+    curve = session.get_curve(name=curve_name)
+    time_series = curve.get_absolute(issue_date_from=date_from, issue_date_to=data_time, data_date=data_time,
+                                     issue_frequency='MIN15')
+    time_series_pandas = time_series.to_pandas()
+    return time_series_pandas
+
+
+def get_instance_data(instance_curves_names: dict, session: Session, issue_date: pd.Timestamp) -> pd.DataFrame:
+    """This function uses the get_instance_curve to retrieve the instance curve data from all benchmark curves, then maps it into a dataframe.
+
+    Args:
+        instance_curves_names: A dictionary containing all instance curve names (the benchmark curves).
+        session: Session object to interact with WAPI.
+        issue_date: The user defined date of issue for the instance curve.
+
+
+    Returns:
+        df: A dataframe containing all instance curve data from the benchmark curves.
+    """
+    df = pd.DataFrame()
```

Comment:
This is basically the exact same method as for the get_absolute_data. Any way to combine these, perhaps?